integrations/tensorflow/e2e/CMakeLists.txt

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Special cases to exclude from automatically expanding targets for all
+# backends.
+# keep sorted
+set(SPECIAL_CASES
+    "mobile_bert_squad_test.py"
+)
+
+# keep sorted
```

Comment:
```suggestion
```

Do we have a CMake formatter / does it parse this like `buildifer` does? This is here in the `BUILD` file just to notify `buildifier` that it should keep these sorted.

Ignore the next few suggestions otherwise.